src/core/src/op/interpolate.cpp

```diff
@@ -454,6 +454,7 @@ bool op::v4::Interpolate::evaluate_interpolate(const HostTensorVector& outputs,
                                                         outputs[0]->get_data_ptr<int8_t>(),
                                                         out_shape,
                                                         m_attrs);
+        break;
```

Comment:
strange that it was not catched by some unit tests

Reply:
@nosovmik can it be related to CVS-67214 ?